src/app/features/task-repeat-cfg/store/task-repeat-cfg.effects.ts

```diff
@@ -58,6 +66,131 @@ export class TaskRepeatCfgEffects {
     { dispatch: false },
   );
 
+  /**
+   * Updates the repeatCfg of a task, if the task was updated.
+   */
+  updateRepeatCfgWhenTaskUpdates: any = createEffect(
```

Comment:
Do we really need this (honestly I am unsure :D)? Are there potential edge cases and problems that this might cause?

Reply:
Not sure about edge cases, but without this you have no way of changing contents of a repeatable task as far as I was able to see.
What I did: I tried to change the title of a repeatable task, but since each day the new tasks are generated based on the repeatCfg (where the title was not updated), the old title would appear.

Reply:
Currently you can change existing repeatable task instances by editing the repeatable config. If you edit the title there, the app asks you if you want to update all existing tasks. The idea behind it is, that you might want to have different notes and even tags for different instances. But I can see how it might be more intuitive to ditch this behavior and do it like this. Let's try it out!